EVB-2/IS_EVB-2/src/wheel_encoder.cpp

```diff
@@ -53,14 +45,15 @@ void step_wheel_encoder(is_comm_instance_t &comm)
 	static int encoderSendTimeMs=0;
 	static wheel_encoder_t wheelEncoderLast = {0};
 
-	if(abs(g_comm_time_ms-encoderSendTimeMs)>=20)
+	if(++encoderSendTimeMs >= 20)
```

Comment:
Can we move the ++encoderSendTimeMs to the line above?  It's easy to miss, but it's important particularly because it's static.  If it was on it's own line it would be more obvious.